docs/application/vstools/tools/certificate-manager.md

```diff
@@ -1,104 +1,254 @@
+       ![Use own certificates](media/vstools_cert_certificate1.png)
 
 3.  Click **OK**.
 
-<a name="creating"></a>
-## Creating a Certificate Profile
+## Create Certificate Profile
 
->**Note**
->
->It is recommended to keep your certificates and password safe in the local repo to prevent it from being compromised.
+If you already have a certificate profile, you can import it or you can create a certificate profile.
 
-A certificate profile consists of an author certificate and 1 or 2 distributor certificates:
+### Import certificate profile
 
--   An author certificate includes information about the author of the application. It is used to create an author signature, which ensures the integrity of the application from the author since the publication of the application.
--   A distributor certificate includes information about the distributor of the application, such as a store. It is used to create a distributor signature, which ensures the integrity of the application from the distributor since the distribution of the application.
+1. In the Visual Studio menu, select **Tools &gt; Tizen &gt; Tizen Certificate Manager**.
+2. To import the profiles.xml file, click **Browse** . 
 
-To distribute your application, you must create a certificate profile and sign the application with it. You can create a new certificate profile with the Certificate Manager:
+    ![Migration for Certificate Profile](media/cert_import_profile.png)
 
-1.  In the Visual Studio menu, select **Tools &gt; Tizen &gt; Tizen Certificate Manager**.
-2.  In the Certificate Manager window, click **+** to create a new profile.
+3. To create a new profile, click **Cancel**. The **Tizen Certificate Manager** window appears.
 
-    ![Certificate Manager window](media/cert_reg_create_new.png)
+    >**Note**
+    >
+    >It is recommended to keep your certificates and password safe in the local repo to prevent it from being compromised.
+
+### Create certificate profile
+
+A certificate profile consists of an author certificate and one or two distributor certificates:
+
+- **Author Certificate** includes information about the author of the application. It is used to create an author signature, which ensures the integrity of the application from the author since the publication of the application.
+- **Distributor Certificate** includes information about the distributor of the application, such as a store. It is used to create a distributor signature, which ensures the integrity of the application from the distributor since the distribution of the application.
+
+To distribute your application, you must create a certificate profile and sign the application with it. To create a certificate profile:
+
+1. Perform step 1 to step 3 under [import certificate profile](#import-certificate-profile).
+2. In the **Tizen Certificate Manager** window, click **App Signing**.
+
+   ![Tizen Certificate Manager window](media/cert_app_signing.PNG)
```

Comment:
Just for the reference. When you capture the images, please use the Windows 10 PC. For now, Windows 10 is the environment guideline for the image capturing. I think you can also check that the updated images are in different UI compared to original images.

Reply:
@mijin85cho,

Sure, we will use Windows 10.